tests/console/gpg.pm

```diff
@@ -32,33 +41,91 @@ sub run {
     wait_still_screen 1;
     type_string "1\n";
     wait_still_screen 1;
-    type_string "2048\n";
+    type_string "$key_size\n";
     wait_still_screen 1;
     type_string "0\n";
     wait_still_screen 1;
     type_string "y\n";
     wait_still_screen 1;
-    type_string "Test User\n";
+    type_string "$user_name\n";
     wait_still_screen 1;
-    type_string "user\@example.com\n", 1;
+    type_string "$email\n", 1;
     type_string "\n";
     wait_still_screen 1;
     save_screenshot;
     type_string "O\n";
 
     assert_screen("gpg-enter-passphrase");
+
     # enter wrong passphrase
     type_string "REALSECRETPHRASE\n";
     assert_screen("gpg-incorrect-passphrase");
     type_string "\t\n", 1;
+
     # enter correct passphrase
-    type_string "R34LS3CR3TPHR4S3\n";
+    type_string "$passwd\n";
     wait_still_screen 1;
-    type_string "R34LS3CR3TPHR4S3\n";
+    type_string "$passwd\n";
     wait_still_screen 1;
-    # list gpg keys
-    validate_script_output("gpg --list-keys", sub { m/\[ultimate\] Test User <user\@example\.com>/ });
+    if (get_var("FIPS") || get_var("FIPS_TS") && $key_size == 1024) {
+        assert_screen("key-generation-failed");
+    }
+    else {
+        # list gpg keys
+        validate_script_output("gpg --list-keys", sub { m/\[ultimate\] $user_name \<$email\>/ });
+    }
+}
+
+sub gpg_encrypt_file {
+
```

Comment:
which empty line should be deleted? is it only under sub or all the empty line which include above of annotations - "#"